server/api/impl/aws/service.go

```diff
@@ -89,8 +92,9 @@ func createAWSConfig(awsAccessKeyID, awsSecretAccessKey string) *aws.Config {
 // NewService creates a new instance of Service.
 func NewService(sess *session.Session) *Service {
 	return &Service{
-		lambda:       lambda.New(sess, aws.NewConfig().WithLogLevel(aws.LogDebugWithRequestErrors)),
```

Comment:
The debug logs are _way_ to noisy. `NewAWSSessionWithLogger` already ensure HTTP requests are logged on `Debug`. I don't think we should rely on the AWS logger and hence I removed this.